README.md

```diff
@@ -1,463 +1,161 @@
 
 
+## About
 
-## About LibreLingo
-
-LibreLingo's mission is to create a modern language-learning platform owned by the community of its users. All software is licensed under AGPLv3, which guarantees the freedom to run, study, share and modify the software. Course authors are encouraged to release their courses with free licenses.
-
- If you want to know why I built LibreLingo, [I recommend reading my article](https://dev.to/kantord/why-i-built-librelingo-280o). If you agree with LibreLingo's mission, [consider helping to build and maintain it](#become-a-contributor), and [please consider making a monetary contribution](https://github.com/sponsors/kantord).
-
-<details open="open">
-  <summary>Table of Contents</summary>
-  <ol>
-    <li>
-      <a href="#about-librelingo">About LibreLingo</a>
-    </li>
-    <li>
-      <a href="#platform">Platform</a>
-      <ul>
-        <li><a href="#web-app">Web app</a></li>
-        <li><a href="#librelingo-yaml-loader">librelingo-yaml-loader</a></li>
-        <li><a href="#librelingo-json-export">librelingo-json-export</a></li>
-      </ul>
-    </li>
-    <li>
-     <a href="#milestones">Milestones</a>
-   </li>
-    <li>
-     <a href="#attributions">Attributions</a>
-   </li>
-    <li>
-     <a href="#articles">Articles</a>
-   </li>
-    <li>
-      <a href="#become-a-contributor">Become a contributor</a>
-    </li>
-    <li>
-      <a href="#see-also">See also</a>
-    </li>
-  </ol>
-</details>
-
-## Platform
-
-<table>
-<tbody>
-<thead>
-<tr>
-<td>
- 
-### Web app
-
-</td>
-</tr>
-</thead>
-<tr>
-<td>
-
-The [LibreLingo web app](https://librelingo.app/) is a lightweight, interactive web app for learning
-foreign languages using LibreLingo courses.
-
-Features:
-* 🎯 practice using interactive exercises
-* 🗓️ spaced repetition
-* 💾 save and monitor progress
-* 🔁 synchronise progress across multiple devices
-* 📱 works on various devices such as phones, tablets and desktop computers
-
-#### Screenshots
- 
-<p align="center">
-    <img src="/docs/screenshots/screenshot1.png" width="15%"></img> 
-    <img src="/docs/screenshots/screenshot2.png" width="15%"></img>   
-    <img src="/docs/screenshots/screenshot3.png" width="15%"></img> 
-    <img src="/docs/screenshots/screenshot4.png" width="15%"></img> 
-</p>
+LibreLingo's mission is to create a modern language-learning platform that is owned by the community of its users. All software is licensed under AGPLv3, which guarantees the freedom to run, study, share, and modify the software. Course authors are encouraged to release their courses with free licenses.
 
+If you want to know why I built LibreLingo, I recommend reading my article. If you agree with LibreLingo's mission, consider helping to build and maintain it, and please consider making a monetary contribution.
 
-</td>
-</tr>
-</tbody>
-</table>
+<b>articles:</b>
```

Comment:
2 issues:

- nit-pick: it feels weird that it's not in title case or at least it doesn't start with a capital letter 
- it should be marked up as a proper heading as opposed to <b></b>
- applies also the rest of the headings, not just this one. It seems unnecessary to end a heading in a colon

Reply:
Okay if it doesn't fit just change it :)
and thanks for informing.

Reply:
fixed